app/src/main/res/values/log_in_dimens.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    
+
```

Comment:
do not use a lot of res files for each project. It's useless. Use one file for all dimensions. Because you are making boilerplate code.

Reply:
done